frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -564,7 +564,7 @@ export const widgets = [
       },
       loadingState: {
         type: 'toggle',
-        displayName: 'Loading State',
+        displayName: 'Loading state',
```

Comment:
Hey @HarshitVashisht11. Please revert this; this is not the requested change. The change is needed in the line number `717`